### DISCUSSION TITLE
[Feature Request] RSocket support as transport layer

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [celery/celery#7526] [Ideas]
https://rsocket.io/
It is a useful alternative to rabbitMQ.

[a comment by an automated account is not shown]

**Comment by auvipy** [comment]
transfering to discussion, also it is not so widely used. our next big plan is to support amqp 1.0

**Comment by LifeIsStrange** [comment]
RSocket support first class communication to web browsers,  which seems to me like a unique value proposition
